pkg/cmd/shell/shell.go

```diff
@@ -0,0 +1,34 @@
+package shell
+
+import (
+	"strings"
+
+	run "github.com/rancher/k3d/cli"
+	"github.com/spf13/cobra"
+
+	"github.com/rancher/k3d/pkg/constants"
+)
+
+type options struct {
+	name  string
+	shell string
+}
+
+func NewCommand() *cobra.Command {
+	opts := &options{}
+	// shellCmd represents the shell command
+	var shellCmd = &cobra.Command{
+		Use:   "shell",
+		Short: "Start a subshell for a cluster",
+		Args:  cobra.MinimumNArgs(0),
```

Comment:
I think we could use some documentation here (i.e. Usage string) to define that we're now expecting args instead of the `--command` flag when directly passing in a command.